docs/guides/hooks-list.rst

```diff
@@ -585,6 +585,14 @@ Other
 	"container_guid" (int) are provided. An empty entity value generally means the form is to
 	create a new object.
 
+**elgg_channels:send, <channel_name>**
```

Comment:
I'd like to keep the API open to basing channels around things other than entities. I could imagine the site entity getting (ab)used for this. I guess the API could let you use give ($entity, $name) or (null, $name) and it would auto-form names like "$guid-$name" and "-name".